src/components/UI/UFormInput/ReactFormInput.tsx

```diff
@@ -0,0 +1,33 @@
+import { RegisterOptions, UseFormRegister } from 'react-hook-form';
+import ISignInFields from '../../../types/sign.in.fields';
+
+interface IReactFormInputProps
```

Comment:
In other places props interfaces don't contain "Props" and here it's contains, why? The naming must be the same throw the app